action-src/main.ts

```diff
@@ -200,7 +204,8 @@ async function run() {
     process.exit(1);
   }
 }
+
 run().catch((e) => {
   error(e);
   setFailed(e.message);
-});
+}).then(() => Sentry.flush(1000));
```

Comment:
Did we want to make this a `.finally` instead?

Reply:
Yep, absolutely.